cmd/mattermost/commands/command_test.go

```diff
@@ -117,11 +117,11 @@ func TestCreateCommand(t *testing.T) {
 			cmds, _ := th.SystemAdminClient.ListCommands(team.Id, true)
```

Comment:
A bit out of scope, but would you also please check the returned `error` here?

Reply:
@hanzei also `ListCommands` returns `([]*Command, *Response)` and not an error

Reply:
I mean checking the `Error` field of `model.Response`.